### PR TITLE
BaSP-MR-197: Forgot password

### PR DESCRIPTION
**Pull request by FontanR**
Now we handle password from all user with firebase.

Password wont be editable from the forms used by any entity.

Instead we handle it from the login as a new functionality with its own page.
This page wil ask for the email, in case the email is not registed in the gym (wont find in firebase db) and a toast with the error will appear.

In case email exists in the db should redirect to login and show a toast telling that a email was sent to the user. Check the email  (check spam) and click on the link given and a new tab asking the new password will appear.

### CODE REVIEW

**Review by jramire5 — COMMENTED** [review]
I cannot edit my password, it keeps showing this toast
image

File: src/Components/Auth/ForgotPassword/index.jsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  const handleSendEmail = (data) => {
+    const auth = getAuth();
+    sendPasswordResetEmail(auth, data.email, { url: `${process.env.REACT_APP_URL}/auth/login` })
+      .then(() => {
+        history.push('/auth/login');
+        dispatch(setContentToast({ message: 'Email with reset link sent', state: 'success' }));
+        dispatch(handleDisplayToast(true));
+      })
+      .catch(() => {
+        dispatch(setContentToast({ message: 'Email is not registed', state: 'fail' }));
```
Comment (jramire5) [inline src/Components/Auth/ForgotPassword/index.jsx:40]: you have a typo here. It´s registered

Comment (FontanR) [reply src/Components/Auth/ForgotPassword/index.jsx:40]: did you add the following code?

Add the following environment variable in the .env file:
REACT_APP_URL=http://localhost:3000/

Comment (jramire5) [reply src/Components/Auth/ForgotPassword/index.jsx:40]: yes! . But now is working, maybe i had to restart the visual studio code

**Review by jramire5 — APPROVED** [review]
LGTM, is working. Good Job

**Review by josedaniele — APPROVED** [review]

**Pull request merged**